swarm/storage/localstore.go

```diff
@@ -184,3 +184,30 @@ func (ls *LocalStore) Iterator(from uint64, to uint64, po uint8, f func(Address,
 func (ls *LocalStore) Close() {
 	ls.DbStore.Close()
 }
+
+func (ls *LocalStore) Migrate() error {
+	schema, err := ls.DbStore.GetSchema()
+	if err != nil {
+		log.Error(err.Error())
+		return err
+	}
+
+	log.Debug("found schema", "schema", schema, "runtime-schema", CurrentDbSchema)
+	if schema != CurrentDbSchema {
+		// run migrations
+
```

Comment:
We should apply all migrations leading from found version to current version. 

```golang
// iterate over versions and apply migrations 
for v < currentVersion {
    if err := Migrations[v](ls); err != nil {
       return err
    }
    v++
    if err := ls.DbStore.PutSchema(v); err != nil {
	return err
    }
}
```

Reply:
This is exactly why I want to use names rather than versions - we would be making implicit assumptions about schemas based on version, that I don't want us it make.

Also this feels like unnecessary complexity right now, when we don't have that many migrations (which we should really be avoiding where possible) - I wouldn't add this now.